src/app/components/register-modal/register-modal.component.html

```diff
@@ -1,66 +1,54 @@
-      <littil-button-rounded
-        data-test="registerButton"
-        [inline]="true"
-        [disabled]="creatingProfile"
-        (click)="onClickRegister()"
-        >Registreren</littil-button-rounded
-      >
-      <littil-button-rounded
-        data-test="cancelButton"
-        [inline]="true"
-        [disabled]="creatingProfile"
-        [color]="'yellow'"
-        (click)="closeModal()"
-        class="mr-4"
-        >Annuleren</littil-button-rounded
-      >
-      <littil-form-error-message *ngIf="registerForm.controls['email'].dirty || registerForm.controls['email'].touched"
-          [errorText]="FormUtil.ContainsError(registerForm.controls['email'], 'required') ? 'E-mailadres is verplicht' : (FormUtil.ContainsError(registerForm.controls['email'], 'email') ? 'Vul een geldig e-mailadres in' : '')"
+  @if(!hideForm){
+  <h1 class="font-bold text-lg text-blue-200 mb-3">Registeren</h1>
+  <p>
+    Vul hier uw e-mailadres in om u te registreren. U ontvangt een e-mail om uw inschrijving te
+    bevestigen en een wachtwoord aan te maken.
+  </p>
+  <p>Na activatie van uw account kunt u uw profiel verder afmaken.</p>
+  <form [formGroup]="registerForm" data-test="form-register" (submit)="onClickRegister()">
+    <littil-form-input-text formControlName="email" id="email" label="E-mailadres">
+      <littil-form-error-message
+        [customErrors]="{
+          required: 'E-mailadres is verplicht'
+        }"
+        controlName="email"
       ></littil-form-error-message>
-    </div>
-    <p *ngIf="creatingProfile">Uw account wordt aangemaakt</p>
-  </ng-container>
-</div>
+    </littil-form-input-text>
+  </form>
 
-<div [@hideShow]="hideConfirmation ? 'hidden' : 'visible'" class="prose">
-  <ng-container *ngIf="!hideConfirmation">
-    <h1 class="font-bold text-lg text-blue-200 mb-3">Registratie gelukt</h1>
-    <p>Check je inbox om in te loggen en een eigen wachtwoord aan te maken.</p>
-    <littil-button-rounded
-      data-test="loginButton"
+  <div class="flex">
+    <littil-button
+      data-test="registerButton"
       [inline]="true"
-      (click)="onClickLogin()"
-      >Inloggen</littil-button-rounded
+      [disabled]="creatingProfile"
+      (click)="onClickRegister()"
+      >Registreren</littil-button
     >
-    <littil-button-rounded
-      data-test="closeButton"
+    <littil-button
+      data-test="cancelButton"
       [inline]="true"
+      [disabled]="creatingProfile"
       [color]="'yellow'"
       (click)="closeModal()"
-      >Sluiten</littil-button-rounded
+      class="mr-4"
+      >Annuleren</littil-button
     >
-  </ng-container>
+  </div>
+  @if(creatingProfile){
+  <p *ngIf="creatingProfile">Uw account wordt aangemaakt</p>
+  } }
+</div>
+
+<div [@hideShow]="hideConfirmation ? 'hidden' : 'visible'" class="prose">
+  @if(!hideConfirmation){
+  <h1 class="font-bold text-lg text-blue-200 mb-3">Registratie gelukt</h1>
+  <p>Check je inbox om in te loggen en een eigen wachtwoord aan te maken.</p>
```

Comment:
Can we change this text.  After registration an email is send with the given password.
There is no option to change your password.

Reply:
Ik heb er dit van gemaakt: "Check je inbox voor een wachtwoord om in te loggen."